### PR TITLE
SEO-2024 Blazor Toast External Linking Changes

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @ChristoIssac ,

Please review the changes made on Blazor Toast External linking.

Regards,
Yvone.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +15,70 @@ To run this application, you need to first clone or download the ‘Create a Toa
...
+## Progress bar
+
+Users can enable a progress bar inside the toast to track the progress of an operation and the expiration of a toast message.
+
+## Toast message validation
+
+Users can validate duplicate toast notifications and allow only distinct toast messages on the screen. They can restrict the maximum number of toasts that are displayed. Also, users can prevent toast messages from hiding and make them stick with the screen.
+
+## Related links
+[Learn More about Blazor Toast](https://www.syncfusion.com/blazor-components/blazor-toast?utm_source=github&utm_medium=listing&utm_campaign=blazor-notifications-github-samples)
```
Comment (Girija-Rajendran) [inline README.md]: invalid UTM param

File: README.md [hunk 2]
```diff
@@ -19,3 +15,70 @@ To run this application, you need to first clone or download the ‘Create a Toa
...
+The Syncfusion's [Blazor components](https://www.syncfusion.com/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-toast-github-samples) library offers over 70 UI components to work with Blazor server-side and client-side (Blazor WebAssembly) projects seamlessly. In addition to Toast, we provide popular Blazor Components such as [DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-buttons-github-samples), [Charts](https://www.syncfusion.com/blazor-components/blazor-charts?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-buttons-github-samples), [Scheduler](https://www.syncfusion.com/blazor-components/blazor-scheduler?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-buttons-github-samples), [Diagram](https://www.syncfusion.com/blazor-components/blazor-diagram?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-buttons-github-samples), and [Word Processor](https://www.syncfusion.com/blazor-components/blazor-word-processor?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-buttons-github-samples).
```
Comment (Girija-Rajendran) [inline README.md]: invalid UTM param
image

File: README.md [hunk 3]
```diff
@@ -19,3 +15,70 @@ To run this application, you need to first clone or download the ‘Create a Toa
...
+
+<hr style="height:0.3px;border:none;color:lightgrey;background-color:lightgrey;" />
+
+<p align="center">
+<a href="mailto:sales@syncfusion.com?Subject=Syncfusion Blazor toast - GitHub" target="_top">sales@syncfusion.com</a> | <a href="https://www.syncfusion.com?utm_source=github&utm_medium=listing&utm_campaign=blazor-toast-github-samples">www.syncfusion.com</a>| Toll Free: 1-888-9 DOTNET <br>
```
Comment (Girija-Rajendran) [inline README.md]: Change this as **Syncfusion Blazor Toast - GitHub**
image

Comment (Yvone-Atieno) [reply README.md]: changed

File: README.md (hunk 2 above)
Comment (Yvone-Atieno) [reply README.md]: UTM parameter changed

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**